tests/validators/literal.test.ts

```diff
@@ -0,0 +1,32 @@
+import { ExpectedValidationError, s } from '../../src';
+
+describe('LiteralValidator', () => {
+	const predicate = s.literal('sapphire');
+
+	test('GIVEN a literal THEN returns the given value', () => {
+		expect(predicate.parse('sapphire')).toBe('sapphire');
+	});
+
+	test("GIVEN anything which isn't the literal THEN throws ExpectedValidationError", () => {
+		expect(() => predicate.parse('hello')).toThrow(
+			new ExpectedValidationError('LiteralValidator', 'Expected values to be equals', 'hello', 'sapphire')
+		);
+	});
+
+	test('GIVEN clone THEN returns similar instance', () => {
+		// @ts-expect-error Test clone
+		const clonePredicate = predicate.clone();
+
+		expect(clonePredicate).toBeInstanceOf(predicate.constructor);
```

Comment:
@kyranet did u mean something like this?

Reply:
Instanceof + strictEquals, so all properties are validated